larky/src/main/java/com/verygood/security/larky/modules/ChaseModule.java

```diff
@@ -0,0 +1,75 @@
+package com.verygood.security.larky.modules;
+
+import com.google.common.collect.ImmutableList;
+import com.verygood.security.larky.modules.vgs.crypto.ChaseCrypto;
+import java.util.List;
+import java.util.ServiceLoader;
+import lombok.SneakyThrows;
+import net.starlark.java.annot.Param;
+import net.starlark.java.annot.ParamType;
+import net.starlark.java.annot.StarlarkBuiltin;
+import net.starlark.java.annot.StarlarkMethod;
+import net.starlark.java.eval.StarlarkBytes;
+
+@StarlarkBuiltin(
+    name = "chase",
+    category = "BUILTIN",
+    doc = "The Chase module contains a few methods for fetching a CA Signed public JWK" +
+          "To serve Chase Bank so they can encrypt the payloads that they send, and to" +
```

Comment:
The "they" is Chase. Chase requires everyone who integrates with them to set up an API endpoint to host a set of public JWK's that Chase will use to encrypt payloads that they send; they also host an endpoint that hosts _their_ public JWK and all payloads sent to Chase are to be encrypted. The integrations being built right now are centered around Chase pushing issued card info to their customers, who are using VGS to remain complaint. 

This PR is here so that VGS customers can call a single function to decrypt Chase's encrypted JWE token, without having access to the private key that can be used for decryption. This is all necessary for any integrations where Chase is the entity sending payloads with PCI data.